src/Core/Checkout/Customer/SalesChannel/CustomerRoute.php

```diff
@@ -33,6 +36,7 @@ public function load(Request $request, SalesChannelContext $context, Criteria $c
         $criteria->setIds([$customer->getId()]);
 
         $customerEntity = $this->customerRepository->search($criteria, $context->getContext())->first();
+        \assert($customerEntity instanceof CustomerEntity);
```

Comment:
we should really teach PHPStan, that the customer is present, if we set the default `_loginRequired(AllowGuest)` :thinking: